components/pd_client/src/errors.rs

```diff
@@ -29,10 +27,22 @@ pub enum Error {
 
 pub type Result<T> = result::Result<T, Error>;
 
+impl Error {
+    pub fn retryable(&self) -> bool {
+        match self {
+            Error::Grpc(_) | Error::Other(_) | Error::ClusterNotBootstrapped(_) => true,
+            Error::RegionNotFound(_)
+            | Error::StoreTombstone(_)
+            | Error::GlobalConfigNotFound(_)
```

Comment:
Why this doesn't need retry?

Reply:
the config should be wrong, it doesn't the result after retry
/cc @glorv

Reply:
`GlobalConfigNotFound` is returned when the target key does not exist. So I think it is reasonable to not retry this error as the retry is likely to return the same error.

Though, In the current implement, the tikv side code just ignores this error with an error log, so there is no chance to propgate this error the outer level error handling code.

https://github.com/tikv/tikv/blob/c461cd829fc6e06f071441f70645c8403e7453a5/components/pd_client/src/client.rs#L309-L318